universum/analyzers/utils.py

```diff
@@ -110,7 +110,15 @@ def expand_files_argument(settings: argparse.Namespace) -> None:
     # TODO: subclass argparse.Action
     result = []
```

Comment:
why not mark result as List[str]?

Reply:
BTW, shouldn't it be set instead of list?

Reply:
Indeed, set seems like a better fit

Reply:
Fixed